src/platform/STM32/timer_def.h

```diff
@@ -1142,7 +1142,7 @@
 
 #elif defined(STM32H7)
 
-#define FULL_TIMER_CHANNEL_COUNT 87
+#define FULL_TIMER_CHANNEL_COUNT 91
```

Comment:
74 entries in master. This PR adds 13 entries. Which would be 87 :joy:

Reply:
Alright I added the missing 4. Some of these definitions are out of order, but probably not worth fixing since we'd have to audit every config.g TIMER_PIN_MAP to ensure they're selecting the right definition

Reply:
With https://github.com/betaflight/betaflight/pull/14153/commits/9908d5215d0fa9843d8f24c631360e29232b847e it is 99